content/src/Blip.Api.Template/MySettings.cs

```diff
@@ -12,6 +14,30 @@ public class MySettings
         /// </summary>
         [JsonProperty("BlipBotSettings")]
         public BlipBotSettings BlipBotSettings { get; set; }
+
+        [JsonProperty("ContentProvider")]
```

Comment:
This should be removed alonside any mention of ContentProviderAPI, and moved to an independent package